fabric/base/docker-compose-base.yaml

```diff
@@ -37,9 +37,9 @@ services:
     working_dir: /opt/gopath/src/github.com/hyperledger/fabric
     command: orderer
     volumes:
-      - ../channel-artifacts/genesis.block:/var/hyperledger/orderer/orderer.genesis.block
-      - ../crypto-config/ordererOrganizations/luther.systems/orderers/orderer0.luther.systems/msp:/var/hyperledger/orderer/msp
-      - ../crypto-config/ordererOrganizations/luther.systems/orderers/orderer0.luther.systems/tls/:/var/hyperledger/orderer/tls
+      - /var/lib/docker/codespacemount/workspace/sandbox/fabric/channel-artifacts/genesis.block:/var/hyperledger/orderer/orderer.genesis.block
```

Comment:
I assume this worked?  I guess we need to expose a var for this then so we don't break fabric locally.

Reply:
Yes, this worked. I guess it doesn't show up in the PR, I marked a couple of the commits (this included) Do Not Submit.